docs/concepts.rst

```diff
@@ -45,7 +45,9 @@ Stream Failure vs. Application Errors
 
 Stream failures are different from *application errors*.  A stream failure indicates that an event stream *cannot* produce more events.  An application error may or may not indicate an event stream failure.
 
-For example, an event stream representing messages arriving on a WebSocket *fails* if the WebSocket is disconnected unexpectedly due to a wifi drop.  In contrast, an application error may occur in the application logic when it receives a WebSocket message that it can't process because the application state has changed.
+For example, an event stream representing messages arriving on a WebSocket *fails* if the WebSocket is disconnected unexpectedly due to a dropped network connection.
```

Comment:
Correct:
> For example, an event stream, representing messages, arriving on a WebSocket *fails* if the WebSocket is disconnected unexpectedly because of a dropped network connection.

"... representing messages ..." is a parenthetical phrase. It adds information that isn't vitally important.
Because of vs. due to: _Due to_ is always used after a form of the verb _to be_. _Because of_ is not used after a form of the verb _to be_.

Reply:
Hmm, making it parenthetical reads a bit awkwardly to me.  What if we rephrase to something like:

"For example, an event stream of WebSocket messages _fails_ if ..."

> Because of vs. due to

:+1: Thanks

Reply:
Or maybe:

"For example, an event stream of messages from a WebSocket fails if ..."

Reply:
Yes. The latter is okay.

Reply:
Updated in 58ac0b4